.travis.yml

```diff
@@ -3,6 +3,7 @@ sudo: false
 branches:
   only:
     - master
+    - version-3
```

Comment:
This will need to be removed _right before merging_. This is needed here so that branches against this `version-3` branch have their tests ran in CI. See https://github.com/webpack-contrib/webpack-bundle-analyzer/pull/143#issuecomment-362899898